src/lazy/encoder/binary/v1_1/value_writer.rs

```diff
@@ -537,10 +532,10 @@ impl<'value, 'top> BinaryValueWriter_1_1<'value, 'top> {
             // 65,792 and higher
             _ => {
                 cold_path! {
-                    xl_symbol_id_value => {
+                    xl_symbol_id_value => {{
```

Comment:
Is the double `{` intentional?

Reply:
It is--the `cold_path!` macro takes a single expression. In order to mark multiple statements as the cold path, you need to pass in a block. The outer `{}` is part of the macro's syntax, the inner `{}` is grouping the statements that follow into a block.